pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -527,6 +552,104 @@ func (exp *flowExporter) addRecordToSet(record flowexporter.FlowRecord) error {
 	return nil
 }
 
+func (exp *flowExporter) addConnToSet(conn *flowexporter.DenyConnection) error {
```

Comment:
Because `Record` and `DenyConnection` have different structures, if we merge the methods, we still need to handle them separately (using if conditions). Two individual methods are clearer to me. Do you have better suggestions?